docs/logging.md

```diff
@@ -27,10 +27,10 @@ To learn how to query, fitler etc. using Kibana you might like to look at this [
 
 You can check to see if any logs are being ingested into Elasticsearch by curling against its URL:
 ```
-$ curl http://130.211.120.118:9200/_search?size=10
+curl -k -u admin:Drt3KdRGnoQL6TQM https://130.211.152.93/api/v1beta1/proxy/services/elasticsearch-logging/_search?size=10
```

Comment:
I think it'd be slightly clearer if you moved that comment up above the wall of text. Your call.